### DISCUSSION TITLE
Allow symbols in tags

### DISCUSSION THREAD

**Discussion by DavinChurch** [aplteam/Tatin#54] [Ideas]
I would like for my package tags to be able to contain `⎕` and other APL characters. For instance, when I want to talk about it enhancing `⎕DT`, that's what I'd like to use in my tags. Likewise, I also wanted to mention that it's related to `1200⌶`, which also requires APL symbology. Other APL characters may likewise be useful in certain circumstances, so I'd prefer that the tags allow symbols of all sorts.

It may be quite reasonable to allow tags to be found without such symbols, but the symbols are important when listing the tags that are provided for a package.

**Comment by aplteam** [comment]
My view is that no APL chars should be allowed, and particularly not the ` ⌶`. These are explicitly experimental, and might eventually be either removed or converted into a `⎕SOMETHING`, meaning they should not be used as tags.

The one exception that might be okay is the `⎕` symbol.

@Morten? @Gil?

**Reply by mkromberg** [reply]
I certainly think it will be natural to use system names like ⎕JSON or ⎕XML in package tags. 1200⌶ is probably due for promotion to a system function very soon. I'm unsure about that one. Whatever decision is made, 1200 must find 1200⌶ and DT must find ⎕DT, I think.

**Comment by e9gille** [comment]
I'm also a bit wary of allowing exotic characters in tags, but maybe that is unfounded. If used in web requests they need to be properly encoded, possibly causing issues, but that's the only reason I have.

**Comment by DavinChurch** [comment]
I think that the quad is probably most important, but I don't see any particular reason why it **must** be restricted. My general opinion on public tools is that they shouldn't artifically restrict things that don't have a good reason to be restricted, if they're useful in some way and compatible with operations.

**Comment by aplteam** [comment]
Rejected. You may include `DT` rather than `⎕DT` if you want to add a tag for the system function `⎕DT`.
